### PR TITLE
Basque language translation

### PR DESCRIPTION
**Pull request by ikusimakusi**
## Description
Adding Basque language translation

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [x] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks 👍  I'll have a look asap an mentioned in the theme PR.

File: translations/actions.eu.xlf [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="eu" datatype="plaintext" original="actiions.en.xlf">
+        <body>
+      <trans-unit id="bEXLcqN" resname="stop">
```
Comment (kevinpapst) [inline translations/actions.eu.xlf]: which tool did you use, that creates these `id` and `resname` attributes?

**Comment by ikusimakusi** [comment]
Thank you very much for your awesome help!! :)

I've fixed the issues you reported on actions.eu.xlf and pagerfanta.eu.xlf

And I also have added the missing translations on messages.eu xlf and teams.eu.xlf.
I've noticed that you reported correctly some of the missing translations but not all of them. For example, it is true that "team.visibility_restricted" was missing, but there were also missing: 
- team.visibility_global
- team.project_visibility_inherited
- team.create_default

Thanks

File: translations/pagerfanta.eu.xlf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="eu" datatype="plaintext" original="none">
+        <body>
+            <trans-unit id="previous">
+                <source>previous</source>
+                <target>aurrekoa</target>
```
Comment (kevinpapst) [inline translations/pagerfanta.eu.xlf]: did you try the result in kimai? these are the translations in the pager below each datable and for my personal preference: I don't like if there is a translation inside (thats why they are empty in all languages):
Bildschirmfoto 2020-01-17 um 12 56 37

**Comment by ikusimakusi** [comment]
Yep, thanks for letting me know.
It looks much nicer without translation text.
I've just committed new changes.
It should be ok now.

**Pull request merged**